src/Analysis/GWASApp/Steps/SelectOutcome/SelectOutcome.jsx

```diff
@@ -4,7 +4,6 @@ import ContinuousCovariates from '../../Components/Covariates/ContinuousCovariat
 import CovariatesCardsList from '../../Components/Covariates/CovariatesCardsList';
 import CustomDichotomousCovariates from '../../Components/Covariates/CustomDichotomousCovariates';
 import ACTIONS from '../../Utils/StateManagement/Actions';
-import './SelectOutcome.css';
```

Comment:
There seem to be some items in this .jsx that still use `className='select-container'`, so should this change be reverted?

Reply:
The class name shouldn't need to be changed, this update just changes the location of the CSS rules. Because the styles are needed in two steps (selecting covariates and selecting outcome) they were moved from SelectOutcome.css to Covariates.css, this is just organizational and shouldn't actually change the appearance of anything.

Reply:
But what about this line
https://github.com/uc-cdis/data-portal/blob/caba1addeb95426c5ab83ac40d4738de88c1abce/src/Analysis/GWASApp/Steps/SelectOutcome/SelectOutcome.jsx#L28
I'm not sure I follow...